notebooks/tutorials/1_getting_started_lesson3/notebook.ipynb

```diff
@@ -0,0 +1,321 @@
+       "      <td>0.333333</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Equity(44060 [ZTS])</th>\n",
+       "      <td>32.68</td>\n",
+       "      <td>0.000000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Equity(44089 [BCC])</th>\n",
+       "      <td>29.66</td>\n",
+       "      <td>1.000000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Equity(44102 [XONE])</th>\n",
+       "      <td>60.50</td>\n",
+       "      <td>0.396667</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Equity(44158 [XOOM])</th>\n",
+       "      <td>27.31</td>\n",
+       "      <td>-0.160000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Equity(44249 [APAM])</th>\n",
+       "      <td>64.53</td>\n",
+       "      <td>0.000000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Equity(44270 [SSNI])</th>\n",
+       "      <td>21.05</td>\n",
+       "      <td>0.423333</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                                close_price  sentiment_score\n",
+       "2013-12-31 00:00:00+00:00 Equity(43721 [SCTY])        57.32        -0.176667\n",
+       "                          Equity(43919 [LMCA])       146.22         0.000000\n",
+       "                          Equity(43981 [NCLH])        35.25        -0.700000\n",
+       "                          Equity(44053 [TPH])         19.33         0.333333\n",
+       "                          Equity(44060 [ZTS])         32.68         0.000000\n",
+       "                          Equity(44089 [BCC])         29.66         1.000000\n",
+       "                          Equity(44102 [XONE])        60.50         0.396667\n",
+       "                          Equity(44158 [XOOM])        27.31        -0.160000\n",
+       "                          Equity(44249 [APAM])        64.53         0.000000\n",
+       "                          Equity(44270 [SSNI])        21.05         0.423333"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Import run_pipeline method\n",
+    "from quantopian.research import run_pipeline\n",
+    "\n",
+    "# Execute pipeline created by make_pipeline\n",
+    "# between start_date and end_date\n",
+    "pipeline_output = run_pipeline(\n",
+    "    make_pipeline(),\n",
+    "    start_date='2013-01-01',\n",
+    "    end_date='2013-12-31'\n",
+    ")\n",
+    "\n",
+    "# Display last 10 rows\n",
+    "pipeline_output.tail(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the next lesson we will formalize the strategy our algorithm will use to select assets to trade. Then, we will use a factor analysis tool to test the performance of our strategy over historical data."
```

Comment:
`to test the performance of our strategy` -> `to evaluate the predictive power of our strategy`